packages/mongo/oplog_v2_converter_tests.js

```diff
@@ -0,0 +1,89 @@
+import { oplogV2V1Converter } from './oplog_v2_converter';
+
+Tinytest.only('oplog - v2/v1 conversion', function(test) {
+  const entry1 = {
+    $v: 2,
+    diff: { scustom: { sEJSON$value: { u: { EJSONtail: 'd' } } } },
+  };
+  const entry2 = {
+    $v: 2,
+    diff: { u: { d: '2', oi: 'asdas' } },
+  };
+  //set inside an array
+  const entry3 = { $v: 2, diff: { sasd: { a: true, u0: 2 } } };
+  //unset inside an array
+  const entry4 = { $v: 2, diff: { sasd: { a: true, u0: null } } };
+
+  //set a new nested field inside an object
+  const entry5 = {
+    $v: 2,
+    diff: { i: { a: { b: 2 } } },
+  };
+
+  //set an existing nested field inside an object
+  const entry6 = {
+    $v: 2,
+    diff: { sa: { i: { b: 3, c: 1 } } },
+  };
+
+  //unset an existing nested field inside an object
+  const entry7 = {
+    $v: 2,
+    diff: { sa: { d: { b: false } } },
+  };
+  const entry8 = { $v: 2, diff: { u: { c: 'bar' }, sb: { a: true, u0: 2 } } };
+
+  const entry9 = {"$v":2,"diff":{"sservices":{"sresume":{"u":{"loginTokens":[]}}}}};
+
+  const entry10 =  {"$v":2,"$set":{"services.resume.loginTokens":[{"when":"2022-01-06T23:58:35.704Z","hashedToken":"RlalW6ZSvPPJLH6sW3B1b+vrUnPy+Ox5oMv3O3S7jwg="},{"when":"2022-01-06T23:58:35.704Z","hashedToken":"DWG0Qw/+nZ48wAIhKR2r9H41wLpth9BM+Br6aZsl2bU="}]}};
+
+  test.equal(
```

Comment:
Why are you not asserting right after the declaration? It's harder to read the test as the declaration is far from the assertation.